EXPLAINER_fullscreen_popups.md

```diff
@@ -0,0 +1,161 @@
+
+Multiple partners and early adopters of the [Multi-Screen Window Placement API](https://w3c.github.io/window-placement/) have specifically requested the ability to show one or more fullscreen popups from a single user gesture, to avoid the user friction entailed with making separate gestures to create each window and make each window fullscreen. For example: 
+*   [window.open should support the 'fullscreen' option](https://github.com/w3c/window-placement/issues/7)
+*   [Feature Request: Initiate a multi-screen experience from a single user activation](https://github.com/w3c/window-placement/issues/98#top)
+*   [Feature request: Fullscreen support on multiple screens](https://github.com/w3c/window-placement/issues/92).
+
+## Goals
+The goal of this document is to outline the necessary spec changes that would permit web applications to open a new fullscreen window on a specific display, with only a single user gesture. It also aims to explore abuse and other security / privacy considerations. In particular this document focuses on introducing algorithmic changes to existing APIs to allow such behavior without introducing API surface changes.
+
+The [Initiating Multi-Screen Experiences Explainer](https://github.com/w3c/window-placement/blob/main/EXPLAINER_initiating_multi_screen_experiences.md) covers some related goals (i.e. launching N fullscreen windows on N displays) not covered in this proposal.
+
+## Use Cases
+Some basic illustrative examples which may be achieved with this proposal are listed below:
+*   Video streaming app launches a video directly to fullscreen on a separate display.
+*   3D modeling app launches a preview in fullscreen on a separate display.
+
+Web applications may already [enter fullscreen on the current window and open a new (non-fullscreen) popup window on a different display with a single user gesture](https://w3c.github.io/window-placement/#usage-overview-initiate-multi-screen-experiences) (due to [this spec text](https://w3c.github.io/window-placement/#api-window-open-method-definition-changes)). By allowing fullscreen popups, this proposal effectively enables web applications to launch multiple fullscreen windows on multiple displays with a single user gesture. Some examples are listed below:
+*   Financial app opens a fullscreen dashboard on the primary monitor and a fullscreen stock tracker window on a secondary display.
+*   Virtual desktop app launches fullscreen windows on primary and secondary displays to mimic a remote monitor configuration.
+
+Furthermore, a web application could launch N fullscreen popup windows, as long as the user agent allows N popups from a user gesture (i.e. multiple popups allowed by popup blocker being disabled):

```

Comment:
This is no longer true, if fullscreen popups consume a user gesture regardless of popup blocker config.

Reply:
See above. Fixed now.